.env.example

```
@@ -0,0 +1,15 @@
+# Set this to either development or production.
+NODE_ENV=production
+# Server info
+HOST=localhost
+PORT=3000
+# Postgres connection string
+POSTGRES_URL="postgresql://USERNAME:PASSWORD@IP:PORT/DBNAME"
```

```suggestion
POSTGRES_URL="postgresql://postgres:postgres@localhost:5432/nin0chat"
```
provide sane defaults instead of placeholders